docs/generate_examples.cmake

```diff
@@ -0,0 +1,48 @@
+
+# Get and update the LUA_PATH so the scripts can be executed without Awesome
+execute_process(COMMAND lua -e print\(package.path\) OUTPUT_VARIABLE LUA_PATH_)
+set(ENV{LUA_PATH} "${SOURCE_DIR}/lib/?;${SOURCE_DIR}/lib/?.lua;${LUA_PATH_}")
+
+execute_process(COMMAND mkdir -p "${BUILD_DIR}/doc/images")
```

Comment:
Can `file(MAKE_DIRECTORY "${BUILD_DIR}/doc/images")` handle this instead (e.g. so that no new process is forked)?